pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - packages/*
   - docs
+  - apps/*
```

Comment:
The "apps" directory seems to create a service that derives from something `@use-funnel`, what about a name like "example"?

Reply:
>I agree, I made it idiomatic without much thought.


I think it would be better to use tests rather than examples. 

Code and examples for e2e testing may initially be similar, but

As testing becomes more sophisticated, it may develop into examples that are too complicated for users.

What do you think?

Reply:
I agree. One of my plans was to include examples of different router adapters, and it would be great to have e2e tests using them. But as you say, if it gets complicated, it would be nice to separate it with a name like "e2e_test"!